src/ua_securechannel.c

```diff
@@ -558,7 +552,7 @@ sendChunkSymmetric(UA_ChunkInfo *ci, UA_Byte **buf_pos, const UA_Byte **buf_end)
         if(channel->securityMode == UA_MESSAGESECURITYMODE_SIGNANDENCRYPT)
             *buf_end -= 2;
     }
-    return ci->errorCode;
+    return UA_STATUSCODE_GOOD;
```

Comment:
Wouldn't returning res be more consistent here, even if it can only be GOOD at the moment? Someone might change something and expect it to return res.

Reply:
done.